WordPress/metadata/PlayStoreStrings.po

```diff
@@ -134,7 +134,7 @@ msgid ""
 "California users privacy notice: https://wp.me/Pe4R-d/#california-consumer-privacy-act-ccpa.\n"
 msgstr ""
 
-#. translators: Title to be displayed in the Play Store. Limit to 50 characters including spaces and commas!
+#. translators: Title to be displayed in the Play Store. Limit to 30 characters including spaces and commas!
```

Comment:
As @jrushfinn pointed out back in January in pcbrnV-440-p2#comment-5319, while the limit used to be 50 characters for the title, Google has since changed it to 30 at some point, but we never updated the comment.

Better make sure our translators are aware of that new smaller limit, especially given they will have to update their translations for the new title during this very sprint.